src/components/ErrorPage.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import './../styles/ErrorPage.css'
+import Navbar from './Navbar'
+import './../styles/App.css'
+
+
+function ErrorPage() {
+    return (
+        <>    <Navbar /> 
+                        
+            <div className="errorPage">
+           
+                
+                <div className="errorPage_content">
+                    <h2 className="errorPage_h2">404 </h2>
+                    <h4 className="errorPage_h4">Oops! Page Not Found</h4>
+                    <p className="errorPage_p">Sorry, the page you were looking for doesn't exist. If you think something is wrong, try again.</p>
+                </div>
+            </div>
+        </>
+    )
+}
+
+export default ErrorPage
```

Comment:
Throughout the application class method has been used.

Reply:
@Aaishpra Make these changes

Reply:
Already done @bismitaguha

Reply:
@bismitaguha is this correct now

Reply:
Yes